pkg/tsdb/grafana-pyroscope-datasource/pyroscopeClient.go

```diff
@@ -190,6 +190,41 @@ func (c *PyroscopeClient) GetProfile(ctx context.Context, profileTypeID, labelSe
 	}, nil
 }
 
+func (c *PyroscopeClient) GetSpanProfile(ctx context.Context, profileTypeID, labelSelector string, spanSelector []string, start, end int64, maxNodes *int64) (*ProfileResponse, error) {
+	req := &connect.Request[querierv1.SelectMergeSpanProfileRequest]{
+		Msg: &querierv1.SelectMergeSpanProfileRequest{
+			ProfileTypeID: profileTypeID,
+			LabelSelector: labelSelector,
```

Comment:
Why is there a labelSelector? I thought the whole point of separate API is that we cannot mix span and label selectors together.